apps/github-stack/src/github-ineraction/test/github-scheduler.spec.ts

```diff
@@ -102,9 +114,9 @@ describe("GithubScheduler", () => {
         stargazers_count: 103,
         watchers_count: 6,
         forks_count: 10509,
-        latestRelease: "v1.7.19",
         repoId: 23,
         user: new User(),
+        releases: [],
```

Comment:
put some releases here. Then ,create mock releas ethat you getFrom gitHub.
And check if there is only one notification sent for a given user.
If there is more than one notification for one repository then something isnt right